docs/integrations.md

```diff
@@ -31,3 +31,12 @@ __Link:__ [https://github.com/adfinis-sygroup/aem-docker/tree/master](https://gi
 __Description:__  "Download and unarchive the latest kompose release asset for your OS"
 
 __Link:__ [https://github.com/chouseknecht/kompose-install-role](https://github.com/chouseknecht/kompose-install-role)
+
+### Fabric8 Maven Plugin by Red Hat
+
+__Description:__  "Maven is one of the widely used build tool for Java applications. The Fabric8 Maven Plugin is a maven extension that simplifies the deployment of Java applications to Kubernetes or OpenShift cluster. The main task of this plugin is to build Docker images, generate Kubernetes or OpenShift resource descriptors and run/deploy the application on Kubernetes or OpenShift cluster. Plugin has wide range of configuration options and Docker Compose is one of them to bring up deployments on Kubernetes or OpenShift cluster. The Fabric8 Maven Plugin processes the external `docker-compose.yml` file and generates Kubernetes or OpenShift resources via Kompose."
+__Links:__
+
+* [Quickstart](/docs/maven-example)
+
+* [Documentation](https://maven.fabric8.io/)
```

Comment:
it will be better if you point towards part of maven docs where `kompose` is being used.
like, `https://maven.fabric8.io/#fabric8:resource`